src/improvements/tasks/eth/000-opcm-upgrade-v200/config.toml

```diff
@@ -0,0 +1,18 @@
+l2chains = [
+    {name = "OP Mainnet", chainId = 10},
+    {name = "Soneium", chainId = 1868},
+    {name = "Ink", chainId = 57073}
+]
+
+templateName = "OPCMUpgradeV200"
+
+[opcmUpgrades]
+absolutePrestates = [
+    {absolutePrestate = "0x039facea52b20c605c05efb0a33560a92de7074218998f75bcdf61e8989cb5d9", chainId = 10},
+    {absolutePrestate = "0x039facea52b20c605c05efb0a33560a92de7074218998f75bcdf61e8989cb5d9", chainId = 1868},
+    {absolutePrestate = "0x039facea52b20c605c05efb0a33560a92de7074218998f75bcdf61e8989cb5d9", chainId = 57073}
+]
+
+[addresses]
+OPCM = "0x026b2F158255Beac46c1E7c6b8BbF29A4b6A7B76"
+StandardValidatorV200 = "0x58b758395be69ada78fe46f9653ff838f1e85251"
```

Comment:
This file needs to contain: 
```
FoundationUpgradeSafe = "0x847B5c174615B1B7fDF770882256e2D3E95b9D92"
SecurityCouncil = "0xc2819DC788505Aac350142A7A707BF9D03E3Bd03"
```
or simulation and signing won't work.

Reply:
15397a1c4034e008e5fc89c9e5136bcffc250944